indigo/apps.py

```diff
@@ -9,3 +16,20 @@ def ready(self):
         # ensure our plugins are pulled in
         import indigo.analysis      # noqa
         import indigo.bulk_creator  # noqa
+
+        # setup sentry error reporting
+        if not settings.DEBUG and os.environ.get('SENTRY_DSN'):
```

Comment:
I think getting the SENTRY_DSN from settings.py, where it is taken from the environ, is better. A single place to store config.

Reply:
Also, can't we do all this in settings.py directly?